Codeinwp/raft
# fix: add logo option to onboarding

### Summary


Updates the Onboarding welcome notice to provide more clarity and adds logo to Onboarding.

### Will affect visual aspect of the product

YES

### Test instructions


- Make sure the Welcome Notice is as expected in the issue.
- Make sure Raft logo is displayed in Onboarding.


Closes https://github.com/Codeinwp/otter-internals/issues/141.
